src/OmniSharp/Api/v1/ProjectSystem/OmnisharpController.ProjectSystem.cs

```diff
@@ -24,11 +25,15 @@ public class ProjectSystemController
 
         [HttpPost("/projects")]
         [HttpGet("/projects")]
-        public WorkspaceInformationResponse ProjectInformation()
+        public WorkspaceInformationResponse ProjectInformation(ProjectInformationRequest request)
         {
+            var includeSourceFiles = true;
+            if (request != null)
```

Comment:
Yeah. We don't check anywhere else.
On 2 Jul 2015 21:29, "jtbm37" <notifications@github.com> wrote:

> In src/OmniSharp/Api/v1/ProjectSystem/OmnisharpController.ProjectSystem.cs
> <https://github.com/OmniSharp/omnisharp-roslyn/pull/256#discussion_r33819427>
> :
>
> >          {
> > +            var includeSourceFiles = true;
> > +            if (request != null)
>
> If clients are already making use of this action method and they do not
> send any parameters it will throw an NRE (NullReferenceException). Are you
> ok with that ?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/OmniSharp/omnisharp-roslyn/pull/256/files#r33819427>.
>